### DISCUSSION TITLE
PyTorch QAT Supported?

### DISCUSSION THREAD

**Discussion by tanyokwok** [pytorch/TensorRT#807] [Q&A]
TensorRT8.0 brings better quantized aware training targeted to Nvidia GPU in PyTorch. However, the toolkits are currently put in the repo: https://github.com/NVIDIA/TensorRT/tree/master/tools/pytorch-quantization.

I think it would be wonderful if Torch-TensorRT would support QAT since the optimization is less robust via onnx.

Is there any progress in PyTorch QAT supported in Torch-TensorRT

**Comment by narendasan** [comment]
You can use QAT in PyTorch and TensorRT today with Torch-TensorRT https://nvidia.github.io/Torch-TensorRT/_notebooks/vgg-qat.html
